src/uct/base/uct_iface.h

```diff
@@ -385,6 +385,7 @@ typedef struct uct_tl_device_resource {
                                               (e.g. UCT_DEVICE_TYPE_NET for a network interface) */
     ucs_sys_device_t         sys_device; /**< The identifier associated with the device
                                               bus_id as captured in ucs_sys_bus_id_t struct */
+    uint64_t                 flags;      /**< Associated flags to the resource */
```

Comment:
remove?

Reply:
we need it to carry the `UCT_TL_RESOURCE_DESC_FLAG_INTER_NODE` flag, to be used in UCP at the time packing for net addresses only.

Reply:
i take it back, you're right, removed

Reply:
double checked agian, this is internal structure populated from `->query_devices()` and not public so we need to keep it